package.json

```diff
@@ -128,6 +128,7 @@
     "@types/react-dom": "16.0.3",
     "@types/react-redux": "5.0.0",
     "@types/request": "2.48.2",
+    "@types/resize-observer-browser": "^0.1.5",
```

Comment:
Is this one ok to use? It seems pretty recent, I think we typically try to keep to supporting safari ~10+? https://caniuse.com/?search=resize%20observer Maybe worth the polyfill though, seems nice

Reply:
I'm adding a guard to just ignore ResizeObserver if not supported.